console/api/main.go

```diff
@@ -154,10 +154,7 @@ func handleRenderAllTemplates() http.HandlerFunc {
 				log.Warningf(ctx, "skipping %q %v", templatePath, err)
 				return nil
 			}
-			if err := generator.Render(f, templatePath, string(tplBytes), def); err != nil {
-				return err
-			}
-			return nil
+			return generator.Render(f, templatePath, string(tplBytes), def)
```

Comment:
This is no longer a linting issue (I committed the PR to remove this check myself) so please let's not do this :)